pkg/manifests/nginx.go

```diff
@@ -444,6 +444,13 @@ func newNginxIngressControllerDeployment(conf *config.Config, ingressConfig *Ngi
 						Name:  "controller",
 						Image: path.Join(conf.Registry, "/oss/kubernetes/ingress/nginx-ingress-controller:"+ingressConfig.Version.tag),
 						Args:  deploymentArgs,
+						Lifecycle: &corev1.Lifecycle{
+							PreStop: &corev1.LifecycleHandler{
+								Exec: &corev1.ExecAction{
+									Command: []string{"/wait-shutdown"},
```

Comment:
can we add a comment here explaining why this is relevant? feel like it's not super obvious, maybe even just a link to nginx docs

Reply:
documented